README.md

```diff
@@ -28,9 +28,9 @@ The Zip distributions can be found on following paths:
 
 ### Execution
 
-* `mvn clean install` - Full build including documentation, automatic QA and maven managed tests. Excludes just Ant and TCK tests. Typical time: 15 minutes.
```

Comment:
Checkstyle is a part of QA. Perhaps would be better to put it into braces after `QA (Checkstyle)`. I am in progress of adding another tool next to the Checkstyle.